src/app/api/teams/overview/route.ts

```diff
@@ -0,0 +1,54 @@
+import { NextRequest, NextResponse } from "next/server";
+import { options } from "@/app/api/auth/[...nextauth]/options";
+import { getServerSession } from "next-auth";
+import { fetchContributionCalculation } from "../../contribution-calculation/fetchContributionCalculation";
+import {
+  fetchTeamContributorsSum,
+  fetchTeamScoreSum,
+} from "../../contributors/team/fetchTeamContributors";
+
+export async function GET(req: NextRequest) {
+  const session = await getServerSession(options);
+  const teamId = req.nextUrl.searchParams.get("team_id");
+
+  if (session?.userId) {
+    const contributionCalculation = await fetchContributionCalculation(teamId!);
```

Comment:
Instead of doing teamId!
just check if on the if
if (session?.userId && teamId)